### ISSUE TITLE
Trigger verify email again after 24h

### ISSUE THREAD

**Issue by sliechti** [sliechti/feedrdr#8]
Then 12h, 6h, warning, fail, disable account.

**Comment by kwigley** [comment]
by disable account.. should I remove user entirely from db ?

**Comment by sliechti** [comment]
No. Let's introduce a flag disabled = true in the users table. I'll change the login to take the flag in consideration.
